content/applications/productivity/discuss/advanced/email_servers.rst

```diff
@@ -0,0 +1,276 @@
+============================================================
+How to Use my Mail Server to Send and Receive Emails in Odoo
+============================================================
+
+If you are a user of Odoo Online or Odoo.sh...
+==============================================
+
+You have nothing to do! **Odoo sets up its own mail servers for your database.** Outgoing and
+incoming emails work out-of-the-box!
+
+Unless you plan to send large batches of mass mailing that could require the use of an external
+mail server, simply enjoy your new Odoo database.
+
+Scope of this documentation
+===========================
+
+This document is **mainly dedicated to Odoo on-premise users** who don't benefit from an
+out-of-the-box solution to send and receive emails in Odoo, unlike
+`Odoo Online <https://www.odoo.com/trial>`__ & `Odoo.sh <https://www.odoo.sh>`__.
+
+.. warning::
+   If no one in your company is used to manage email servers, we strongly recommend that you opt
+   for those Odoo hosting solutions. Their email system works instantly and is monitored by
+   professionals. Nevertheless you can still use your own email servers if you want to manage your
+   email server's reputation yourself.
+
+You will find here below some useful information on how to integrate your own email solution with
+Odoo.
+
+.. note:: Office 365 email servers don't easily allow to send external emails from hosts like Odoo.
+   Refer to `Microsoft's documentation <https://support.office.com/en-us/article/How-to-set-up-a-mu
+   ltifunction-device-or-application-to-send-email-using-Office-365-69f58e99-c550-4274-ad18-c805d65
+   4b4c4>`__ to make it work.
+
+How to manage outbound messages
+===============================
+
+As a system admin, go to :menuselection:`Settings --> General Settings` and check
+:guilabel:`External Email Servers`. Then, click :guilabel:`Outgoing Mail Servers` to create one and
+reference the SMTP data of your email server. Once all the information has been filled out, click
+on :guilabel:`Test Connection`.
+
+Here is a typical configuration for a G Suite server.
+
+.. image:: email_servers/outgoing_server.png
+   :align: center
+   :alt: Connection information for a G Suite server
+
+Then set your email domain name in the General Settings.
+
+.. note::
+   If you get a ```[AUTHENTICATIONFAILED] Invalid credentials (Failure)``` warning when you *Test
+   Connection* on a Gmail address, activate the :guilabel:`Less secure app access` option. A direct
+   link can be `accessed here <https://myaccount.google.com/lesssecureapps?pli=1>`_.
```

Comment:
it's better to use password app imo

Reply:
```suggestion
   If you get a ```[AUTHENTICATIONFAILED] Invalid credentials (Failure)``` warning when you *Test
   Connection* on a Gmail address, create an :guilabel:`app password` in Google. A direct
   link can be `accessed here <https://support.google.com/accounts/answer/185833?hl=en>`_.
```